### ISSUE TITLE
Create docs for installing on a travis agent

### ISSUE THREAD

**Issue by nbgooch** [fsaintjacques/semver-tool#57]
I would use this, but since there are no install docs, I will go elsewhere..

**Comment by ranger6** [comment]
Looks like at least one person is blocked by lack of documentation.  It's true that the installation process is not clearly documented.  As well, the dependencies for use or development are not listed.  Note that reading the Makefile pretty much covers this. However, some users might just want to install the tool and consider the Makefile as part of a "build".

I'd suggest that the `README.md` include a section to cover this.

As for the "installing on a travis agent" reference:

1.  I assume that "travis agent" means a particular travis build environment/configuration.  I could not find "travis agent" searching the internet.
2.  It's pretty clear that we can't document all the different installation processes that might exist in the world, travis included.  I believe the relevant information is in [Installing Dependencies](https://docs.travis-ci.com/user/installing-dependencies#installing-projects-from-source) .

@nbgooch , if you are blocked: here is the short answer: `semver` is a bash shell script.  You need bash in your environment.  You can install `semver` by copying `src/semver` to anywhere you wish, be sure the execute permissions are set, you may want to insure it can be found on your `PATH`.  You can even rename it to whatever you wish.

**Comment by ranger6** [comment]
@nbgooch seems to be ok as we've not gotten any feedback from the previous conversation.  As well, over the past several months, the README installation instructions have been improved and extended.  So, I think that the claim that "there are no install docs" is less valid.

I marked this as "wontfix" and after a reasonable period of time without effort to move this issue forward, it needs to be retired.

**Comment by ranger6** [comment]
Spring cleaning!  See above discussion.
